File: aula03-managed-point-cs/Ponto.cs
using System; // <=> import java.lang;

public class Ponto{

	public readonly int  _y, _z, _x;

	public Ponto(int x, int y, int z)
	{
		this._x = x;
		this._y = y;
		this._z = z;
	}

	public double GetModule() {	 
		return Math.Sqrt((double)_x*_x + _y*_y + _z*_z);
	}

	public void Print(){
		String msg = String.Format("Version 4 Point (x = {0}, y = {1}, z = {2})", _x, _y, _z);
        Console.WriteLine(msg);
	}

}

File: aula03-managed-point-java/Ponto.java
public class Ponto{

	public final int  _y, _z, _x;

	public Ponto(int x, int y, int z)
	{
		this._x = x;
		this._y = y;
		this._z = z;
	}

	public double getModule() {	 
		return Math.sqrt((double)_x*_x + _y*_y + _z*_z);
	}

	public void print(){
		String msg = String.format("Version 4 Point (x = %d, y = %d, z = %d)", _x, _y, _z);
        System.out.println(msg);
	}

}